class.osc.edu/apps/bc_osc_rstudio_server/form.js

```diff
@@ -131,6 +136,23 @@ function alert_if_one_version(one_version) {
   }
 }
 
+function set_cluster(event) {
+  var node_type = 'any';
+  var cluster = 'owens';
+
+  k8s_classrooms.forEach(cls => {
+    var k8s = RegExp(cls).test(event.target.value);
+
+    if(k8s) {
+      node_type = 'owens';
+      cluster = 'kubernetes';
+    }
+  });
```

Comment:
@treydock here's why I'm asking about production clusters on test.  This is client side javascript, so adding kubernetes-test (and the toggle for it) would be difficult.

Reply:
We figured that the best solution is a regular expression test on the URL of the browser to toggle to different k8s clusters. It actually isn't that difficult, especially since all the hosts are known.